antora/package.json

```diff
@@ -26,6 +26,8 @@
   },
   "dependencies": {
     "@antora/collector-extension": "^1.0.0-alpha.3",
-    "@antora/lunr-extension": "^1.0.0-alpha.8"
+    "@antora/lunr-extension": "^1.0.0-alpha.8",
+    "fs": "^0.0.1-security",
+    "toml": "^3.0.0"
```

Comment:
Could go to devDependencies, we're not shipping this we only need it at build time